qunit-dom-codemod.js

```diff
@@ -20,6 +20,32 @@ export default function(file, api, options) {
     },
   };
 
+  const assertEqualBoolean = {
+    callee: {
+      type: 'MemberExpression',
+      object: { name: 'assert' },
+      property: { name: 'equal' },
+    },
+    arguments(args) {
+      return args.length >= 2 && j.match(args[1], {
+        type: 'Literal',
+        value: v => typeof v === 'boolean'
+      });
+    }
+  };
+
+  function assertTruthy(p) {
```

Comment:
can we rename this to `assertBoolean`? `assertTruthy` implies to only check for `true`, but this is also checking for `false`, right?

Reply:
Ah right, makes sense! Fixed!